### DISCUSSION TITLE
Extended regulon on eRegulon metadata

### DISCUSSION THREAD

**Discussion by yurasong** [aertslab/scenicplus#373] [Q&A] [answered]
Hi!
Thanks for the nice tool :)

I would like to ask regarding the eRegulon metadata, which I can get with `format_egrns()` function.
There are regulons which is including _extended_ in the name of signature, which is not clear how I should interpret the term. The example is as below. I already check databases which is used for SCENIC+ but did not get clear definition.

- Chd2_+_-_(533g)
- Chd2_extended_+_-_(579g)

Thanks in advance!

**Comment by SeppeDeWinter** [comment, marked as answer]
Hi @yurasong 

"extended" in this case means that the eRegulon is based on motifs that are annotated to TFs non-directly, this is either based on orthology or based on motif similarity. The other eRegulons (without "extended") are based on motifs that are annotated directly to the TF (for example ChIP-seq for the TF has been performed in the same species as your data and that particular motif was discovered in that ChIP-seq experiment).

I hope this helps?

All the best,

Seppe

**Reply by yurasong** [reply]
Thanks @SeppeDeWinter That's clear for me :)

Best,
Yura

**Reply by SeppeDeWinter** [reply]
You are welcome :)
